apps/files_sharing/lib/Controller/ShareAPIController.php

```diff
@@ -651,7 +651,8 @@ private function getSharesInDir(Node $folder): array {
 		$resharingRight = false;
 		$known = [];
 		foreach ($shares as $share) {
-			if (in_array($share->getId(), $known) || $share->getSharedWith() === $this->currentUser) {
+			if (in_array($share->getId(), $known)
+				|| ($share->getSharedWith() === $this->currentUser && $share->getShareType() === IShare::TYPE_USER)) 
```

Comment:
```suggestion
				|| ($share->getSharedWith() === $this->currentUser && $share->getShareType() === IShare::TYPE_USER)) {
```

Have you tested this code? 😉

Reply:
?

Reply:
> ?

There was actually a `{` missing in the code which causes even the PHP-CS to fail hard.

Reply:
it's still missing

Reply:
fixed